Library/Formula/rdup.rb

```diff
@@ -1,9 +1,9 @@
 class Rdup < Formula
   desc "Utility to create a file list suitable for making backups"
-  homepage "https://archive.miek.nl/projects/rdup/"
-  url "https://archive.miek.nl/projects/rdup/rdup-1.1.14.tar.bz2"
-  sha256 "b25e2b0656d2e6a9cb97a37f493913c4156468d4c21cea15a9a0c7b353e3742a"
-  revision 2
+  homepage "https://miek.nl/"
```

Comment:
Or should it just be https://github.com/miekg/rdup now since there appears to no longer be any rdup specific info at https://miek.nl/ ?